ebi_eva_common_pyutils/command_utils.py

```diff
@@ -11,16 +11,19 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+import logging
 import subprocess
+from select import select
+from threading import Thread
```

Comment:
Don't think we are using **select** and **threading** in the file

Reply:
Removed